src/model.py

```diff
@@ -349,6 +375,45 @@ async def generate(self, request):
             raise e
         finally:
             self.ongoing_request_count -= 1
+            
+    def verify_loras(self, requests):
+        verified_requests = []
+        for request in requests:
```

Comment:
I'm not a fan of this for loop, since we'll loop through requests twice in this case. How about we only keep the body of this loop in a helper function, then in execute we'll use it as:
```python
def execute(self, requests):
    for request in requests:
        request = self.verify_loras(request)
        self.create_task(self.generate(request))
    return None
```

But then we'd need to handle a case when we got an error and sent an error already. In this case maybe we'd need to return a `None` from helper function and hadle it inside `execute`, i.e. :
```python
def execute(self, requests):
    for request in requests:
        request = self.verify_loras(request)
        if request is not None:
            self.create_task(self.generate(request))
    return None
```